api/types/server.go

```diff
@@ -485,16 +501,34 @@ func (s *ServerV2) openSSHEC2InstanceConnectEndpointNodeCheckAndSetDefaults() er
 	return nil
 }
 
+// serverNameForEICE returns the deterministic Server's name for an EICE instance.
+func serverNameForEICE(s *ServerV2) string {
+	awsAccountID := s.GetAWSAccountID()
+	awsInstanceID := s.GetAWSInstanceID()
+
+	if awsAccountID != "" && awsInstanceID != "" {
+		return fmt.Sprintf("%s-%s", awsAccountID, awsInstanceID)
```

Comment:
Does the instance ID always include the `i-` part?

Reply:
It should, yes.
We don't validate it but AWS is always returning the `i-` part as far as I can tell.